requirements-test.txt

```diff
@@ -7,4 +7,4 @@ twine
 pytest-mock
 torch
 # Pin scikit-learn
-scikit-learn<1.1.2
+# scikit-learn<1.1.2
```

Comment:
I assume this change is because sklearn is already in requirements.txt?

Reply:
I was trying to test if pin was required. Looks like it is still.

Reply:
Removed this change.